src/services/web3/config.ts

```diff
@@ -4,6 +4,8 @@ import { Token } from 'services/observables/tokens';
 import { calculatePercentageChange } from 'utils/formulas';
 import { get7DaysAgo } from 'utils/pureFunctions';
 import { EthNetworks } from './types';
+import { BigNumber } from 'bignumber.js';
+import { BigNumberish } from 'ethers';
```

Comment:
I see BigNumberish imported but no new code of it used

Reply:
Fixed